### PR TITLE
Fastroping: deployment API and waypoint

### PR DESCRIPTION
**Pull request by BaerMitUmlaut**
**When merged this pull request will:**
- Add `ace_fastroping_fnc_deployAI`, a public API function for deploying all units that sit in the cargo of a helicopter.
- Add a waypoint which does the same automatically.

Requires #3553.

### CODE REVIEW

File: addons/fastroping/functions/fnc_deployAI.sqf [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+ * Public: Yes
+ */
+
+#include "script_component.hpp"
+params [["_vehicle", objNull, [objNull]], ["_deploySpecial", false, [true]], ["_createDeploymentGroup", true, [true]]];
+private ["_config", "_configEnabled", "_deployTime", "_unitsToDeploy", "_deployGroup"];
+
+if (isNull _vehicle || {!(_vehicle isKindOf "Helicopter")}) exitWith {
+    if (hasInterface) then {
+        ["deployAI was called with an invalid or non-existant vehicle.", QFUNC(deployAI)] spawn BIS_fnc_guiMessage;
```
Comment (BaerMitUmlaut) [inline addons/fastroping/functions/fnc_deployAI.sqf]: Note: `BIS_fnc_guiMessage` causes a CTD with call.

Comment (Glowbal) [inline addons/fastroping/functions/fnc_deployAI.sqf]: Add comment in the code please

Comment (esteldunedain) [inline addons/fastroping/functions/fnc_deployAI.sqf]: Do we really want to show a clickable error message if somebody places a waypoint of this type by mistake?

Comment (BaerMitUmlaut) [inline addons/fastroping/functions/fnc_deployAI.sqf]: I usually use `BIS_fnc_error`, but it doesn't work in Eden.

Comment (esteldunedain) [inline addons/fastroping/functions/fnc_deployAI.sqf]: So this message would only show in the editor, or may it happen too during the mission?

I'd prefer it didn't show during the mission, as it's probably non-critical (plus players can't do anything about it)

**Comment by jonpas** [comment]
Tagging as WIP until that fix is PRed and merged, so no one merges it by mistake.

**Comment by esteldunedain** [comment]
@BaerMitUmlaut, does #3553 contain the fix you are referring to here?

**Comment by BaerMitUmlaut** [comment]
Yes, b20e433 is the fix to be precise.

**Pull request merged**